src/main/java/JigaChat.java

```diff
@@ -0,0 +1,153 @@
+import java.util.Scanner;
+public class JigaChat {
+    Task[] taskList = new Task[100];
+    int taskCounter = 0;
+
+    public void printCommandList() {
+        printTodoCommand();
+        printDeadlineCommand();
+        printEventCommand();
+        printListCommand();
+        printMarkCommand();
+        printUnmarkCommand();
+    }
+
+    private static void printUnmarkCommand() {
+        System.out.println("unmark [task index]                         Marks a task in your list as not done");
+    }
+
+    private static void printMarkCommand() {
+        System.out.println("mark [task index]                           Marks a task in your list as done");
+    }
+
+    private static void printListCommand() {
+        System.out.println("list                                        Displays a list of all tasks and their statuses as well as types");
+    }
+
+    private static void printEventCommand() {
+        System.out.println("event [description] /from [from] /to [to]   Adds a new event to your list");
+    }
+
+    private static void printDeadlineCommand() {
+        System.out.println("deadline [description] /by [by]             Adds a new deadline to your list");
+    }
+
+    private static void printTodoCommand() {
+        System.out.println("todo [description]                          Adds a new todo to your list");
+    }
+
+    public void readCommand() {
```

Comment:
this function is a little long (> 30LOC) try separating it into multiple functions 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods